### PR TITLE
Blog refactor

### PR DESCRIPTION
**Pull request by ryan-blunden**
Mainly markup changes to rely more on Bootstrap styles. As a result, the blog index page displays thumbnails consistently and is more mobile-friendly.

### CODE REVIEW

File: website/src/pages/go.tsx [hunk 1]
```diff
@@ -4,23 +4,23 @@ import * as React from 'react'
...
+    //     const hubspot = document.getElementById('gopherConForm')
+    //     hubspot.appendChild(script)
+    //     script.addEventListener('load', () => {
+    //         ;(window as any).hbspt.forms.create({
+    //             portalId: '2762526',
+    //             formId: '9cfe9111-fe45-45f5-873a-9f4925affdfd',
+    //             target: '#gopherConForm',
+    //         })
+    //     })
+    // }
```
Comment (rvantonder) [inline website/src/pages/go.tsx]: Is there a reason this is commented out but not deleted?

File: website/src/pages/strange-loop.tsx [hunk 2]
```diff
@@ -4,23 +4,23 @@ import * as React from 'react'
...
+    //     const hubspot = document.getElementById('strangeLoopForm')
+    //     hubspot.appendChild(script)
+    //     script.addEventListener('load', () => {
+    //         ;(window as any).hbspt.forms.create({
+    //             portalId: '2762526',
+    //             formId: 'b4b2b911-0515-4a76-bfc0-2f4ca384bdbc',
+    //             target: '#strangeLoopForm',
+    //         })
+    //     })
+    // }
```
Comment (rvantonder) [inline website/src/pages/strange-loop.tsx]: Flagging as commented out.

**Comment by rvantonder** [comment]
I remember some overflow issues with blog posts on mobile and it's possible to reproduce. Don't know if this PR is meant to solve it, so just flagging since you asked me to have a look. 

Example, see release blog post 3.4:

Screen Shot 2020-03-30 at 9 22 59 PM

File: website/src/pages/go.tsx (hunk 1 above)
Comment (ryan-blunden) [reply website/src/pages/go.tsx]: Because it will be re-used once (if) we host the liveblog for GopherCon again.

Comment (ryan-blunden) [reply website/src/pages/go.tsx]: I'll fix.

File: website/src/pages/strange-loop.tsx (hunk 2 above)
Comment (ryan-blunden) [reply website/src/pages/strange-loop.tsx]: Since removed.

**Comment by ryan-blunden** [comment]
> I remember some overflow issues with blog posts on mobile and it's possible to reproduce.

Thanks for this, I couldn't remember which post it was that caused the issue.  To fix that post would require content changes so I'm ok with this PR not addressing that.

**Pull request merged**